### PR TITLE
AA-264 mfe courseware reset dates

### PR DESCRIPTION
**Pull request by xnick421**
- add toast display after successful reset of dates via banner.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@edx/frontend-component-footer": "^10.0.6",
     "@edx/frontend-component-header": "^2.0.3",
     "@edx/frontend-platform": "^1.5.1",
-    "@edx/paragon": "^7.2.1",
+    "@edx/paragon": "^9.1.1",
```
Comment (mikix) [inline package.json:40]: The max width change in 8.0 seems big -- the frontend looks OK after this upgrade?

**Review by mikix — APPROVED** [review]
Sure

**Review by Dillon-Dumesnil — APPROVED** [review]
Sure.  Before updating Paragon, make sure there is approval from TNL as well.

**Comment by xnick421** [comment]
Screen Shot 2020-08-06 at 9 21 32 AM
Screen Shot 2020-08-06 at 1 43 07 PM

File: src/toast/LearningToast.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}) {
+  const {
+    displayResetDatesToast,
+  } = useSelector(state => state.courseHome);
+  const dispatch = useDispatch();
+
+  return (
+    displayResetDatesToast && (
+      <Toast
+        onClose={() => dispatch(toggleResetDatesToast({ displayResetDatesToast: false }))}
```
Comment (ciduarte) [inline src/toast/LearningToast.jsx]: I'd pull these into the parent so we can reuse this component for styling.

File: src/course-home/data/__snapshots__/redux.test.js.snap [hunk 3]
```diff
@@ -103,6 +105,7 @@ Object {
   "courseHome": Object {
     "courseId": "course-v1:edX+DemoX+Demo_Course_1",
     "courseStatus": "loaded",
+    "displayResetDatesToast": false,
```
Comment (ciduarte) [inline src/course-home/data/__snapshots__/redux.test.js.snap:108]: I see updated snapshots but I'd double check the tests/factories to see if there's anything you can add there.

**Review by ciduarte — APPROVED** [review]

**Pull request merged**